services/postgres/seed.ts

```diff
@@ -7,42 +7,104 @@ async function main() {
+        { id: "21", name: "eth-mainnet", weight: 1, params: "" },
+        { id: "22", name: "eth-archival", weight: 1, params: "" },
+        { id: "28", name: "eth-trace", weight: 1, params: "" },
+        { id: "62", name: "ethereum-mainnet-high-gas", weight: 1, params: "" },
+        { id: "46", name: "evmos-mainnet", weight: 1, params: "" },
+        { id: "49", name: "fantom-mainnet", weight: 1, params: "" },
+        { id: "5", name: "fuse-mainnet", weight: 1, params: "" },
+        { id: "000A", name: "fuse-archival", weight: 1, params: "" },
+        { id: "27", name: "gnosischain-mainnet", weight: 1, params: "" },
+        { id: "000C", name: "gnosischain-archival", weight: 1, params: "" },
+        { id: "26", name: "eth-goerli", weight: 1, params: "" },
+        { id: "63", name: "goerli-archival", weight: 1, params: "" },
+        { id: "40", name: "harmony-0", weight: 1, params: "" },
+        { id: "44", name: "iotex-mainnet", weight: 1, params: "" },
+        { id: "71", name: "kava-mainnet", weight: 1, params: "" },
+        { id: "72", name: "kava-mainnet-archival", weight: 1, params: "" },
+        { id: "56", name: "klaytn-mainnet", weight: 1, params: "" },
+        { id: "24", name: "poa-kovan", weight: 1, params: "" },
+        { id: "57", name: "meter-mainnet", weight: 1, params: "" },
+        { id: "58", name: "metis-mainnet", weight: 1, params: "" },
+        { id: "50", name: "moonbeam-mainnet", weight: 1, params: "" },
+        { id: "51", name: "moonriver-mainnet", weight: 1, params: "" },
+        { id: "52", name: "near-mainnet", weight: 1, params: "" },
+        { id: "47", name: "oKc-mainnet", weight: 1, params: "" },
+        { id: "70", name: "oasys-mainnet", weight: 1, params: "" },
+        { id: "69", name: "oasys-mainnet-archival", weight: 1, params: "" },
+        { id: "53", name: "optimism-mainnet", weight: 1, params: "" },
+        { id: "54", name: "osmosis-mainnet", weight: 1, params: "" },
+        { id: "1", name: "mainnet", weight: 1, params: "" },
+        { id: "9", name: "poly-mainnet", weight: 1, params: "" },
+        { id: "000B", name: "poly-archival", weight: 1, params: "" },
+        { id: "000F", name: "polygon-mumbai", weight: 1, params: "" },
+        { id: "74", name: "polygon-zkevm-mainnet", weight: 1, params: "" },
+        { id: "23", name: "eth-ropsten", weight: 1, params: "" },
+        { id: "75", name: "scroll-testnet", weight: 1, params: "" },
+        { id: "6", name: "solana-mainnet", weight: 1, params: "" },
+        { id: "60", name: "starknet-mainnet", weight: 1, params: "" },
+        { id: "61", name: "starknet-testnet", weight: 1, params: "" },
+        { id: "76", name: "sui-mainnet", weight: 1, params: "" },
+        { id: "03CB", name: "avax-cra", weight: 1, params: "" },
+        { id: "67", name: "velas-mainnet", weight: 1, params: "" },
+        { id: "68", name: "velas-mainnet-archival", weight: 1, params: "" },
```

Comment:
agree with weight format!

not sure about separating our id with pokt-id, we don't have any offering outside pokt yet? would you be fine if `id was integer` and `current id turns into poktId`?

Reply:
I think the id should be autogenerated like others, and pokt-id seems reasonable. My point wasn't that we offer anything outside pokt, though we should consider this use case. The point was more that we might want multiple aliases to the same pokt-id. This might not be important though, but something that is at least possible.